test/e2e/components/vl-grid.js

```diff
@@ -1,9 +1,170 @@
 const { VlElement } = require('vl-ui-core').Test;
 
+class VlRegion extends VlElement {
+    constructor(driver, selector) {
+        super(driver, selector);
+    }
+
+    async hasNoSpace() {
+        return this._hasClass('no-space');
+    }
+
+    async hasNoSpaceBottom() {
+        return this._hasClass('no-space-bottom');
+    }
+
+    async hasNoSpaceTop() {
+        return this._hasClass('no-space-top');
+    }
+
+    async isAlt() {
+        return this._hasClass('alt');
+    }
+
+    async isSmall() {
+        return this._hasClass('small');
+    }
+
+    async isMedium() {
+        return this._hasClass('medium');
+    }
+
+    async hasBorder() {
+        return this._hasClass('bordered');
+    }
+
+    async _hasClass(name) {
+        return this.hasClass(`vl-region--${name}`);
+    }
+}
+
+class VlLayout extends VlElement {
+    constructor(driver, selector) {
+        super(driver, selector);
+    }
+}
+
 class VlGrid extends VlElement {  
     constructor(driver, selector) {
         super(driver, selector);
     }
+
+    async isStacked() {
+        return this._hasClass('is-stacked');
+    }
+
+    async isHorizontallyStartAligned() {
+        return this._hasClass('align-start');
+    }
+
+    async isHorizontallyCenterAligned() {
+        return this._hasClass('align-center');
+    }
+
+    async isHorizontallyEndAligned() {
+        return this._hasClass('align-end');
+    }
+
+    async isHorizontallyAlignedWithSpaceBetween() {
+        return this._hasClass('align-space-between');
+    }
+
+    async isHorizontallyAlignedWithSpaceAround() {
+        return this._hasClass('align-space-around');
+    }
+
+    async isVerticallyTopAligned() {
+        return this._hasClass('v-top');
+    }
+
+    async isVerticallyCenterAligned() {
+        return this._hasClass('v-center');
+    }
+
+    async isVerticallyBottomAligned() {
+        return this._hasClass('v-bottom');
+    }
+
+    async isVerticallyStretched() {
+        return this._hasClass('v-stretch');
+    }
```

Comment:
ik zie in de component dat dat maar "gewoon" CSS classes zijn, en dat de component ook niet automatisch nog classes toevoegt of zo, en dat die classes dus al door de afnemer gekozen zijn. 
is het dan wel heel nuttig om daar testen/"page object methodes" voor te voorzien? volgens @SanderKleykens op jouw vraag @coemans ?